Freshli.Web/Migrations/20200831222448_AddIdentityTables.cs

```diff
@@ -0,0 +1,248 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+using Npgsql.EntityFrameworkCore.PostgreSQL.Metadata;
+
+namespace Freshli.Web.Migrations
+{
+    public partial class AddIdentityTables : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AlterColumn<string>(
+                name: "url",
+                table: "analysis_requests",
+                nullable: false,
```

Comment:
Interesting. The `[Required]` attribute added to the `Url` field in the `AnalysisRequest` class appears to have been picked up by the migration and is now enforced by the database. Not a problem. Just thought I'd point it out as something worth noting.